lib/model.js

```diff
@@ -1596,10 +1596,10 @@ Model.prototype.findOrInitialize = Model.prototype.findOrBuild = function(option
  * However, it is not always possible to handle this case in SQLite, specifically if one transaction inserts and another tries to select before the first one has comitted. In this case, an instance of sequelize.TimeoutError will be thrown instead.
  * If a transaction is created, a savepoint will be created instead, and any unique constraint violation will be handled internally.
  *
- * @param {Object}   options
- * @param {Object}   options.where where A hash of search attributes.
- * @param {Object}   [options.defaults] Default values to use if creating a new instance
- *
+ * @param {Object}      options
+ * @param {Object}      options.where where A hash of search attributes.
+ * @param {Object}      [options.defaults] Default values to use if creating a new instance
+ * @param {Transaction} [options.transaction] Transaction to run query under
```

Comment:
Oh, I missed it.
I think this edit make easier for untrained English speakers to find it.

Anyway I will add instruction to "see above" in description of `option.transaction`.